lib/knife-solo/bootstraps.rb

```diff
@@ -63,10 +63,10 @@ def gem_packages
       def http_client_get_url(url, file)
         stream_command <<-BASH
           /bin/sh -c " \
-            if command -v curl >/dev/null 2>&1; then \
-              curl -L -o '#{file}' '#{url}'; \
+            if command -v sudo curl >/dev/null 2>&1; then \
```

Comment:
This checks for the presence of sudo rather than curl, so probably not what we want to do here.